sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -663,6 +663,11 @@ def ParsePaths(paths):
     return tokens
 
 
+def create_scope_from_url(url):
+    # Need a better way to do this, regex?
+    return url.replace(":443", "") + ".default"
```

Comment:
If we actually do a urlparse we should be able to reconstruct the URL with the extra ".default" path and without the port?